tweetapi/registration.py

```diff
@@ -36,21 +36,20 @@ class Meta:
         model=User
         fields=("username","password")
 
-class ChangeSerializers(serializers.ModelSerializer):
-    password = serializers.CharField(write_only=True, required=True)
-    new_password = serializers.CharField(write_only=True, required=True,validators=[validate_password])
-    password_confirmation = serializers.CharField(write_only=True, required=True)
+class ChangeSerializers(serializers.Serializer):
+    password = serializers.CharField(write_only=True)
+    password_confirmation = serializers.CharField(write_only=True)
     class Meta:
-        model = User
         fields = (
-            'password','password_confirmation','new_password'
+            'password','password_confirmation'
         )
     def validate(self, attrs):
         user =self.context.get('user')
```

Comment:
look into `require_context`

Reply:
ok sir

Reply:
sir so i checked for required_contex and no article had this well explained, not even on the DRF documentations, still looking for a better Materia for it.

Reply:
I can't find it as well. I must have forgotten where I saw it